smart-contracts/contracts/cl-vault/src/vault/admin.rs

```diff
@@ -80,7 +49,8 @@ pub fn execute_update_admin(
     ADMIN_ADDRESS.save(deps.storage, &new_admin)?;
 
     Ok(Response::new()
-        .add_attribute("action", "execute_update_admin")
+        .add_attribute("method", "execute")
```

Comment:
Why do we use method execute? Is not every call a method execute?

Reply:
yes, indeed. initially i was comparing it to a REST API interface where the `method` of an HTTP call is either GET, POST, PUT or DELETE which executes a controller handler (`action`) that can be a custom endpoint

In this context the "method" was meant to refer the entrypoint we are executing, that could be `instantiate`, `execute`, or `migrate`. like to indicate the specific entrypoint invoked

i havent yet applied this to `instantiate` and `migrate` so if this approach sounds complex i can either remove "method" or "action", or follow the same pattern for the remaining ones

Reply:
Imo, it would be better to adopt a split here in terms of "location" in the contract, so method can be exact_deposit and action can be create_position or something